index.js

```diff
@@ -3,6 +3,8 @@
 const assert = require('assert');
 const util = require('util');
 
+const deprecate = require('depd')('bem-cell');
```

Comment:
And what is purpose of this module here?
why is it better than just `console.warn('modName: just for compatibility and can be dropped in future')` ?